packages/help-center/src/components/help-center-article.tsx

```diff
@@ -38,64 +34,46 @@ export const HelpCenterArticle: React.FC = () => {
 				}
 			}, 0 );
 		}
-	}, [ articleUrl, post ] );
+	}, [ postUrl, post ] );
 
+	// Handle redirecting to new routes
```

Comment:
I suppose this cancels the need for https://github.com/Automattic/wp-calypso/pull/92899? cc @agrullon95.

Reply:
Yes, that's correct. I spoke with Tony and tested these changes. This will fix https://github.com/Automattic/wp-calypso/pull/92899. I will rename that PR and mainly use it to rename the navigation logic to make it clearer.